src/pathfinding_service/service.py

```diff
@@ -256,8 +257,16 @@ def handle_channel_closed(self, event: ReceiveChannelClosedEvent) -> None:
 
         log.info("Received ChannelClosed event", event_=event)
 
-        token_network.handle_channel_closed_event(channel_identifier=event.channel_identifier)
-        self.database.delete_channel(event.token_network_address, event.channel_identifier)
+        channel = self.database.get_channel(event.token_network_address, event.channel_identifier)
```

Comment:
Getting the channel just to delete it feels wrong. Maybe do 
```
if self.database.delete_channel(event.token_network_address, event.channel_identifier):
```
instead and allow deleting 0 or 1 channel? Then we wouldn't need the get_channel code, anymore.

But this is ok, too.

Reply:
I found another problem and fixed that now.

I'll merge this as is to get the PFS to work for the nightly scenarios. But let's discuss this tomorrow again.